src/main/java/frc/robot/commands/climb/WinchesForward.java

```diff
@@ -0,0 +1,92 @@
+package frc.robot.commands.climb;
+
+import edu.wpi.first.wpilibj.command.Command;
+import frc.robot.RobotMap;
+import frc.robot.subsystems.Climber;
+
+
+public class WinchesForward extends Command {
+    public WinchesForward() {
+        // Use requires() here to declare subsystem dependencies
+        // eg. requires(chassis);
+    }
+
+
+    /**
+     * The initialize method is called just before the first time
+     * this Command is run after being started.
+     */
+    @Override
+    protected void initialize() {
+        if(Climber.getInstance().isEngaged()){
+            Climber.getInstance().setWinchLeftMaster(RobotMap.CLIMBER.WINCH_POWER);
+            Climber.getInstance().setWinchRightMaster(RobotMap.CLIMBER.WINCH_POWER);
+        }
+    }
+
+
+    /**
+     * The execute method is called repeatedly when this Command is
+     * scheduled to run until this Command either finishes or is canceled.
+     */
+    @Override
+    protected void execute() {
```

Comment:
If you're not using a method, you can (should) omit the empty definition to make the file more concise.